src/Uno.UI/Uno.UI.csproj

```
@@ -38,35 +38,48 @@
 	<Import Project="..\Uno.CrossTargetting.props" />
 
 	<ItemGroup>
-		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml;" Visible="False" />
+	<!-- The excluded files can be restored as the types are made available in Uno. 
+		Note: the NavigationView style can be restored once ItemsRepeater is available. https://github.com/unoplatform/uno/issues/3192
+		Note: The ProgressRing style can be restored once determinate mode is implemented. https://github.com/unoplatform/uno/issues/3604 -->
+		<WinUIStylePage Include="Themes\WinUI\Resources\**\*.xaml" Exclude="Themes\WinUI\Resources\PriorityDefault\RadioButtons.xaml;Themes\WinUI\Resources\PriorityDefault\RadioButtons_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\RadioMenuFlyoutItem_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\TabView.xaml;Themes\WinUI\Resources\PriorityDefault\TabView_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\TeachingTip.xaml;Themes\WinUI\Resources\PriorityDefault\TeachingTip_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\NavigationView.xaml;Themes\WinUI\Resources\PriorityDefault\ProgressRing.xaml;" />
```

Let's put this in a excluded list, similar to this:
```suggestion
    <ExcludedStyles Include="Themes\WinUI\Resources\PriorityDefault\RadioButtons.xaml" />
    <ExcludedStyles Include="Themes\WinUI\Resources\PriorityDefault\RadioButtons_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\RadioMenuFlyoutItem_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\TabView.xaml;Themes\WinUI\Resources\PriorityDefault\TabView_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\TeachingTip.xaml;Themes\WinUI\Resources\PriorityDefault\TeachingTip_themeresources.xaml;Themes\WinUI\Resources\PriorityDefault\NavigationView.xaml;Themes\WinUI\Resources\PriorityDefault\ProgressRing.xaml" />
		<WinUIStylePage Include="Themes\WinUI\Resources\**\*.xaml" Exclude="@(ExcludedStyles)" />
```

---

Sure! I'll create a separate PR for that, so that this one can be merged and tested.